app/components/content_card.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Image from 'next/image';
+
+interface contentCardProps {
+    type?: contentCardType,
+    title: string,
+    text: string,
+    className?: string,
+    delimiter?: string,
+    path?: string,
+    xPosition?: string,
+    yPosition?: string,
+}
+
+enum contentCardType {
```

Comment:
nit: It is nice to name enums with PascalCase.

```suggestion
enum ContentCardType {
```